doc/source/user_guide/ug_implementation.rst

```diff
@@ -452,7 +452,9 @@ block equally.  This is useful in POP which always has work in
 each block and is written with a lot of
 array syntax requiring calculations over entire blocks (whether or not
 land is present).  This option is provided in CICE as well for 
-direct-communication compatibility with POP. The ‘latitude’ option 
+direct-communication compatibility with POP. Block that contain 100%
+land grid cells are eliminated with 'block'.  The 'blockall' option is identical
```

Comment:
I don't understand the sentence that begins with 'Block'.

Reply:
I fixed it, it should read "Blocks that contain 100% land grid cells are eliminated with 'block'.  The 'blockall' option is identical to 'block' except it does not do land block elimination."  Let me know if that's still not clear.